### PR TITLE
updated with fa 19 final projects

### PR DESCRIPTION
**Pull request by RizkyWellyanto**
I added the final projects for FA 2019. Also added a script to populate the markdowns of the project

### CODE REVIEW

File: _projects/FA19-TableTop_WorldBuilder.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (andrewipark) [inline _projects/FA19-TableTop_WorldBuilder.md]: No download for this one

File: _projects/FA19-Physics_Sandbox.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+layout: project
+title: Physics Sandbox
+desc: A semi-open world environment where a player can freely traverse, spawn in ﻿various physics-affected﻿ objects﻿﻿﻿﻿﻿﻿﻿﻿﻿﻿﻿﻿﻿﻿﻿﻿﻿, and bend the laws of physics to their liking.
+dl: https://drive.google.com/file/d/1ord-pZklNSXUS2G_Cnua54sQ5uM-e05u/view?usp=sharing
+semester: Fall 2019
+team: 24
+youtube: 8thxlS-uSfk
+featured: false
```
Comment (andrewipark) [inline _projects/FA19-Physics_Sandbox.md]: this should be featured

File: _projects/FA19-Food_Nutrition_VR_Game.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+layout: project
+title: Food Nutrition VR Game
+desc: It is an application that is a collection of  3 minigames.  The first two teach the user about basic nutrition, while the third is a fun  arcade shooter.
```
Comment (andrewipark) [inline _projects/FA19-Food_Nutrition_VR_Game.md]:
```suggestion
desc: It is an application that is a collection of 3 minigames. The first two teach the user about basic nutrition, while the third is a fun arcade shooter.
```

(nit, style)

File: _projects/FA19-AR_Accountancy_App.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+layout: project
+title: AR Accountancy App
+desc: It is an ios App that allows you to scan an object and acquire its information in AR. This project is based on ARkit 3 by Apple and developed using Xcode 11 with Swift 5.
```
Comment (andrewipark) [inline _projects/FA19-AR_Accountancy_App.md]:
```suggestion
desc: It is an iOS app that allows you to scan an object and acquire its information in AR. This project is based on ARKit 3 by Apple and developed using Xcode 11 with Swift 5. 
```
(nit, style) capitalization

File: _projects/FA19-60s_Main_Quad.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
+layout: project
+title: 60s Main Quad
+desc: A vritual tour back to 1960s UIUC's main quad.
```
Comment (andrewipark) [inline _projects/FA19-60s_Main_Quad.md]:
```suggestion
desc: A virtual tour back to UIUC's main quad in the 1960s.
```

File: _projects/FA19-AR_Obstacle_Navigation.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+---
+layout: project
+title: AR Obstacle Navigation
+desc: This is the AR implementation of Professor Hernandez' Obstacle Navigation test. It is designed to receive treadmill data in order to avoid objects. The goal was to "step" on top of certain blocks while avoiding others. Its goal is to appear as if the blocks were in
```
Comment (andrewipark) [inline _projects/FA19-AR_Obstacle_Navigation.md]:
```suggestion
desc: This is the AR implementation of Professor Hernandez' Obstacle Navigation treadmill test. The project uses data from the treadmill to test user's abilities to step on or avoid floor obstacles.
```
This was my group, so I feel fairly comfortable suggesting this.

File: _projects/FA19-AR_Obstacle_Navigation.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+---
+layout: project
+title: AR Obstacle Navigation
+desc: This is the AR implementation of Professor Hernandez' Obstacle Navigation test. It is designed to receive treadmill data in order to avoid objects. The goal was to "step" on top of certain blocks while avoiding others. Its goal is to appear as if the blocks were in 
+dl: https://drive.google.com/open?id=1C456N9-r94LcqR1BvgcIfr9gYfSNIaPG
+semester: Fall 2019
+team: 255
```
Comment (andrewipark) [inline _projects/FA19-AR_Obstacle_Navigation.md]:
```suggestion
team: 20
```

File: _projects/FA19-A_day-100_Sec-_as_a_Farmer.md [hunk 8]
```diff
@@ -0,0 +1,10 @@
+---
+layout: project
+title: A day(100 Sec) as a Farmer
```
Comment (andrewipark) [inline _projects/FA19-A_day-100_Sec-_as_a_Farmer.md]:
```suggestion
title: A day (100 sec) as a Farmer
```

File: _projects/FA19-Amusement_Park.md [hunk 9]
```diff
@@ -0,0 +1,10 @@
+---
+layout: project
+title: Amusement Park
+desc: This is a FPS game that simulates amusement park. The player sits in a cart which will move automatically inside the playground. There are two scenes in this game with opposite themes. The first scene is fairy tale scene in which the player will have increasing goal for each level. The second scene is the horror scene. If the player shoot 5 targets then he/she will get level up and targets start to move. If the player is caught by any targets, then GAME OVER...
```
Comment (andrewipark) [inline _projects/FA19-Amusement_Park.md]:
```suggestion
desc: This is a FPS game that simulates an amusement park. The player sits in a cart which will move automatically inside the playground. There are two thematically opposite scenes. The first is a fairy tale scene, and the second is a horror scene. If the player shoot 5 targets then he/she will get level up and targets start to move. If the player is caught by any targets, then GAME OVER...
```

File: _projects/FA19-Ant-Man_Simulator.md [hunk 10]
```diff
@@ -0,0 +1,10 @@
+---
+layout: project
+title: Ant-Man Simulator
+desc: Inspired by Marvel’s Ant Man, this simulator allows players can explore a house different perspectives (grow and shrink between 3 different sizes). In this version, the player has to collect four different coins to win the game, completing a couple of puzzles along the way.
```
Comment (andrewipark) [inline _projects/FA19-Ant-Man_Simulator.md]:
```suggestion
desc: Inspired by Marvel’s Ant Man, this simulator allows players to explore a house in different perspectives by growing and shrinking between 3 different sizes. In this version, the player has to collect four different coins to win the game, completing a couple of puzzles along the way.
```

File: _projects/FA19-Job_Site_Risk_Assessment_-_Hazards_and_risks_of_injury_in_auto_industry.md [hunk 11]
```diff
@@ -0,0 +1,10 @@
+---
+layout: project
+title: Job Site Risk Assessment / Hazards and risks of injury in auto industry
```
Comment (andrewipark) [inline _projects/FA19-Job_Site_Risk_Assessment_-_Hazards_and_risks_of_injury_in_auto_industry.md]:
```suggestion
title: Job Site Risk Assessment / Hazards and Risks of Injury in Auto Industry
```

File: _projects/FA19-VR_for_Studying_Spatial_and_Temporal_Perception_.md [hunk 12]
```diff
@@ -0,0 +1,10 @@
+---
+layout: project
+title: VR for Studying Spatial and Temporal Perception 
+desc: This is a sponsored project. This project is two experiments that take place in a city  environment. The first experiment is has the "player" walk down a straight corridor,  once the player hits an intersection, they input how many seconds they perceive it  took to travel down the street. The second experiment was supposed to track the path  of player and check if he would take signs to move in certain directions or not.  Our sponsor did not give us the detail for the signs, so the player can  only free roam around the city.
```
Comment (andrewipark) [inline _projects/FA19-VR_for_Studying_Spatial_and_Temporal_Perception_.md]:
```suggestion
desc: This is a sponsored project. This project is two experiments that take place in a city environment. The first experiment is has the "player" walk down a straight corridor,  once the player hits an intersection, they input how many seconds they perceive it took to travel down the street. The second experiment was supposed to track the path  of player and check if he would take signs to move in certain directions or not. Our sponsor did not give us the detail for the signs, so the player can only free roam around the city. 
```

File: _projects/FA19-TableTop_WorldBuilder.md (hunk 1 above)
Comment (RizkyWellyanto) [reply _projects/FA19-TableTop_WorldBuilder.md]: Uh... there's 'dl' below with the download link. What are you referring to?

Comment (andrewipark) [reply _projects/FA19-TableTop_WorldBuilder.md]: I thought this was the one we shouldn't have a download for. Let's ask the professors

Comment (RizkyWellyanto) [reply _projects/FA19-TableTop_WorldBuilder.md]: Ahh got it 👍

**Pull request merged**